lib/_emerge/Scheduler.py

```diff
@@ -1521,9 +1521,9 @@ def _build_exit(self, build):
             self.curval += 1
             merge = PackageMerge(merge=build, scheduler=self._sched_iface)
             self._running_tasks[id(merge)] = merge
-            if (
-                not build.build_opts.buildpkgonly
-                and build.pkg in self._deep_system_deps
+            if not build.build_opts.buildpkgonly and (
+                "merge-wait" in build.settings.features
+                or build.pkg in self._deep_system_deps
             ):
                 # Since dependencies on system packages are frequently
                 # unspecified, merge them only when no builds are executing.
```

Comment:
Yeah, now updated to refer to merge-wait.

I also made the _system_merge_started part only apply to system packages, since that's only used for _unsatisfied_system_deps tracking which is actually a separate feature from the merge-wait thing.

Reply:
(taking into account the current change)

Maybe:

```
# By default, merge-wait only allows merge when no builds are executing.
# As a special exception, dependencies on system packages are frequently
# unspecified and will force merge-wait.
```

Reply:
Thanks, updated with your comment.